### ISSUE TITLE
Backup error while upgrading to 4.0.0

### ISSUE THREAD

**Issue by rpelissi** [thorsten/phpMyFAQ#3284]
**Describe the bug**
When trying to upgrade from phpmyfaq 3.2.x to 4.0.0 I receive a "unknow error" during the update/backup step

**To Reproduce**

1. Have phpmyfaq 3.2.x installed and configured
2. Download phpmyfaq 4.0.0
3. Move phpmyfaq folder to a new folder
4. unzip the 4.0.0 archive
5. copy the config folder to the new phpmyfaq folder
6. copy the data folder to the new phpmyfaq folder
7. fix the permissions
8. access to <site url>/update
9. start the update process

**Expected behavior**
The backup should have been done or and more detail message should be displayed

**Screenshots**
![image](https://github.com/user-attachments/assets/ae6b926d-3ff2-4530-a22a-44ec6adf7e1f)

**phpMyFAQ (please complete the following information):**

- phpMyFAQ version: 3.2.x
- PHP version: 8.2
- Database: MariaDB 10.6.19
- Elasticsearch yes/no: no

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Windows 11
- Browser [e.g. chrome, safari]: Firefox
- Version [e.g. 22]: 133.03

**Additional context**
I have enable the debug option but no more information displayed..

**Comment by thorsten** [comment]
@rpelissi despite the error message, did the update went through?

**Comment by rpelissi** [comment]
Thanks!!
So, the admin part seems to be ok but not the main page like you can see on the screenshots:
![image](https://github.com/user-attachments/assets/5d9a5983-7cb7-46fa-9f6e-c3fff7577475)
![image](https://github.com/user-attachments/assets/d3797c2b-6add-477d-b739-fb543651ad7a)

**Comment by rpelissi** [comment]
On the admin/Users section I got this:


```
phpMyFAQ Fatal error

Uncaught exception: 'Twig\Error\RuntimeError'

Message: 'An exception has been thrown during the rendering of a template ("Undefined array key "permission::add_section"").'

Stack trace:

#0 /www/html/phpmyfaq/src/libs/twig/twig/src/Template.php(349): Twig\Template->yield()
#1 /www/html/phpmyfaq/src/libs/twig/twig/src/Template.php(364): Twig\Template->display()
#2 /www/html/phpmyfaq/src/libs/twig/twig/src/TemplateWrapper.php(35): Twig\Template->render()
#3 /www/html/phpmyfaq/admin/user.php(191): Twig\TemplateWrapper->render()
#4 /www/html/phpmyfaq/admin/index.php(269): require('...')
#5 {main}

Thrown in '/www/html/phpmyfaq/assets/templates/admin/user/users.twig' on line 179
```

**Comment by thorsten** [comment]
Looks like not all update queries went through as

`permission::add_section`

was removed in 4.0

**Comment by thorsten** [comment]
But I'll add a check to avoid the error

**Comment by rpelissi** [comment]
Also, I have logoff and try to login again with my ldap account and it does not work anymore it seems.

**Comment by thorsten** [comment]
@rpelissi please create a new issue for the LDAP issue

**Comment by thorsten** [comment]
> Thanks!! So, the admin part seems to be ok but not the main page like you can see on the screenshots: ![image](https://private-user-images.githubusercontent.com/385483/394792567-5d9a5983-7cb7-46fa-9f6e-c3fff7577475.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5MzYwMjYsIm5iZiI6MTczMzkzNTcyNiwicGF0aCI6Ii8zODU0ODMvMzk0NzkyNTY3LTVkOWE1OTgzLTdjYjctNDZmYS05ZjZlLWMzZmZmNzU3NzQ3NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMVQxNjQ4NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MTU4MmY4ZGRhNzgzOTU5MWUzYTU1ZjJlYjQ3NzQxYTlkZDQyYTBiYzhmOTQ2ODBhNjM5N2VkNGIzNWJhMTZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.GH0AUIfkEYQs15X_gdtsDA6HzARB_284jYW0TKFTkCs) ![image](https://private-user-images.githubusercontent.com/385483/394792656-d3797c2b-6add-477d-b739-fb543651ad7a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5MzYwMjYsIm5iZiI6MTczMzkzNTcyNiwicGF0aCI6Ii8zODU0ODMvMzk0NzkyNjU2LWQzNzk3YzJiLTZhZGQtNDc3ZC1iNzM5LWZiNTQzNjUxYWQ3YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMVQxNjQ4NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04N2ZhMzAzZGU0ZTYyZTA4NTczNjc1NjgwODE2YTIwYTA0MjI2MjkyZGMyOWY5ZjNiZmE1MGYxNzUzZDI2ZGE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.swl5XyDgzklTTtQOPMiNktCoFWp2pPvHDaSa9n3ANLQ)

The frontend looks like a caching error.

**Comment by thorsten** [comment]
Fixed all issues for the 4.0.1 release.
